scripts/fuzz_shell.js

```diff
@@ -165,17 +189,36 @@ try {
 // Handle the exports.
 var exports = instance.exports;
 
-var view;
+// Link in a second module, if one was provided.
+if (secondBinary) {
+  var secondModule = new WebAssembly.Module(secondBinary);
 
-// Recreate the view. This is important both initially and after a growth.
-function refreshView() {
-  if (exports.memory) {
-    view = new Int32Array(exports.memory.buffer);
+  // Merge the original imports object with the exports from the first module.
+  // Note that, sadly, Object.assign does not work on wasm exports.
+  var combinedImports = Object.assign({}, imports);
+  combinedImports['primary'] = {};
+  for (var e in exports) {
+    combinedImports['primary'][e] = exports[e];
+  }
```

Comment:
This can just be something like `var combinedImports = {'primary': exports};`. See for example https://github.com/emscripten-core/emscripten/blob/bdde3cd337ed4ac72a641f862b5371048e049946/src/preamble.js#L686.

Reply:
Oh, thanks, I didn't realize wasm-split routes all secondary module imports through the primary module... does that not add any inefficiency? I guess not as the primary module just imports and exports them, unchanged.

Reply:
Right, I don't think it should have any overhead.